conf/routes

```diff
@@ -10,4 +10,11 @@ GET     /                           controllers.HomeController.index
 GET     /assets/*file               controllers.Assets.versioned(path="/public", file: Asset)
 
 # GitHub API
-GET     /api/user/:username     controllers.ApplicationController.getGitHubUser(username)
\ No newline at end of file
+GET     /api/user/:username     controllers.ApplicationController.getGitHubUser(username)
+
+
+# Data Repo Crud Endpoints
+POST    /api/create                        controllers.ApplicationController.create()
```

Comment:
Best practice - don't include CRUD verbs in the route. Instead, just write 'user'.
Example:
GET /api/user/:username      ...